client/src/app/site/motions/modules/motion-detail/components/motion-title-change-recommendation/motion-title-change-recommendation.component.scss

```diff
@@ -0,0 +1,11 @@
+::ng-deep {
```

Comment:
if possible, try to avoid shadow piercing. In your case, you will most likely do not need it.
See the other comments